src/xkey.vsc

```diff
@@ -11,4 +11,25 @@
 
 	Number of surrogate keys in use. Increases after a request that includes a new key in the xkey header. Decreases when a key is purged or when all cache objects associated with a key expire.
 
+.. varnish_vsc:: g_hashhead_bytes
```

Comment:
please give us a little bit more info than the one-liner. Which one grows when, is there any relation between them, etc. In short, as a user how are they useful to me?